game/scripts/player/player.gd

```diff
@@ -33,6 +37,9 @@ func _physics_process(delta):
 		var acc: float = (weapon_force / mass)
 		velocity += Vector2(last_direction.x * acc, last_direction.y * acc)
 	move_and_slide()
+	
+	# Open inventory menu if action pressed
+	inventory_input() 
```

Comment:
I think this is fine for now :+1 When we work on the player entity in earnest we'll probably want a more generalizable  way to look through inputs